karma.conf.ci.js

```diff
@@ -33,16 +33,6 @@ var customLaunchers = {
     browserName: 'safari',
     version: '9.0'
   },
-  sl_ie_7: {
```

Comment:
SauceLabs removed IE7 and Karma doesn't support IE8 anymore so I removed them. Looks like we've already done this for other projects segmentio/isodate@83aac34393561512b25a75e2f25a096d5cd87dcd